test_data.rb

```diff
@@ -448,52 +443,93 @@ def test_sncf_virtual_station
     end
   end
 
-  AIRPORT_TRANSLATIONS = {
-    :fr => 'aeroport',
-    :en => 'airport',
-    :de => 'flughafen',
-    :es => 'aeropuerto',
-    :it => 'aeroporto',
-    :nl => 'luchthaven',
-    :da => 'lufthavn',
-    :ru => 'аэропорт'
-  }
-
   def test_airports_comments_and_children
-    STATIONS.select do |row|
+    # Check that no airport station mention "airport" in the comments, and that all children are also airports
+    SUGGESTABLE_STATIONS.select do |row|
       row['is_airport'] == 't'
     end.each do |row|
-      # Disabled while info are not cleaned yet
-      # assert (AIRPORT_TRANSLATIONS.none? do |locale, translation|
-      #   comment = slugify(row["info:#{locale}"] || '')
-      #   comment =~ /#{translation}/i
-      # end), "One or more comments for #{row['name']} (#{row["id"]}) are mentionning an airport which is not needed as this stations is flagged as an airport"
+      refute has_localized_info?(row, AIRPORT_TRANSLATIONS),
+        "One or more comments for #{row['name']} (#{row["id"]}) are mentionning an airport which is not needed as this stations is flagged as an airport"
 
-      # Check that all children are also airports
       CHILDREN[row['id']].each do |child_row|
-        assert child_row['is_airport'] == 't', "#{child_row['name']} (#{child_row['id']}) should be an airport, as a child of airport station #{row['name']} (#{row['id']})"
+        assert child_row['is_airport'] == 't',
+          "#{child_row['name']} (#{child_row['id']}) should be an airport, as a child of airport station #{row['name']} (#{row['id']})"
       end
     end
+
+    # Look for station including an airport translation in their name that should be flagged as airport
+    SUGGESTABLE_STATIONS.select { |row| row['is_airport'] == 'f' }.each do |row|
+      refute has_localized_info?(row, AIRPORT_TRANSLATIONS),
+        "Station #{row['name']} (#{row["id"]}) contains a mention to 'airport' in its name. This stations should be flagged as an airport instead"
+    end
   end
 
   def test_country_hints_and_children
-    STATIONS.select do |row|
+    SUGGESTABLE_STATIONS.select do |row|
       row['country_hint'] == 't'
     end.each do |row|
-      # TODO once info are cleaned: check that the comment does not contain country information
+      # Check that the comment does not contain country information
```

Comment:
Indeed, but it would require including some locales, which seems a bit overkill isn't it?